src/Components/Artwork/Metadata.tsx

```diff
@@ -14,7 +14,7 @@ export interface ArtworkMetadataProps extends React.HTMLProps<ArtworkMetadata> {
 
 export class ArtworkMetadata extends React.Component<ArtworkMetadataProps, null> {
   static defaultProps = {
-    extended: false,
+    extended: true,
```

Comment:
This default makes more sense, we usually always show the sale/contact line, and can be overridden by passing in false.